### PR TITLE
DNM : Testing using a dummy PR

### PR DESCRIPTION
**Pull request by GomathiselviS**
Depends-On: https://github.com/ansible/ansible-zuul-jobs/pull/422

### CODE REVIEW

**Comment by pabelanger** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

File: zuul.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+- project:
+    check:
+      jobs:
+        - ansible-test-network-unit-openvswitch-python27
```
Comment (pabelanger) [inline zuul.yaml:5]: this needs to update to
```
---
- project:
    check:
      jobs:
        - ansible-test-network-unit-openvswitch-python27:
            vars:
              ansible_test_collections: true
```

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
rebase

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Pull request closed without merging**